packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -1,10 +1,117 @@
-import { shallowMount } from '@vue/test-utils';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import { VueI18n } from '@justeat/f-globalisation';
 import VRating from '../Rating.vue';
+import i18n from './helpers/setup';
+
+const localVue = createLocalVue();
+localVue.use(VueI18n);
 
 describe('Rating', () => {
+    let propsData;
+    let wrapper;
+    const $tc = jest.fn();
+
+    beforeEach(() => {
+        propsData = {
+            starRating: 2,
+            maxStarRating: 5
+        };
+        wrapper = shallowMount(VRating, {
+            propsData,
+            localVue,
+            i18n,
+            mocks: {
+                $tc
+            }
+        });
+    });
+
     it('should be defined', () => {
-        const propsData = {};
-        const wrapper = shallowMount(VRating, { propsData });
         expect(wrapper.exists()).toBe(true);
     });
+
+    describe('methods', () => {
+        describe('`hasRating`', () => {
+            it('should exist', () => {
+                expect(wrapper.vm.hasRating).toBeDefined();
+            });
+
+            it('should contain a description `c-rating-description`', () => {
+                // Act
+                const result = wrapper.find('[data-test-id="c-rating-description"]');
+
+                // Assert
+                expect(result).toMatchSnapshot();
+            });
+
+            describe('when invoked', () => {
+                it('should return truthy when `starRating` is less than or equal to `rating`', () => {
```

Comment:
Should we add a test checking when `starRating` is less than rating? I only see a test for equal here

Reply:
I think the current test covers this? If the code was modified and the `=` removed the test would fail in that instance.